client/tests/integration/smartcontracts/executor_custom_data_model/src/simple_isi.rs

```diff
@@ -23,7 +23,9 @@ pub struct MintAssetForAllAccounts {
     pub quantity: Numeric,
 }
 
-impl From<CustomInstructionBox> for Custom {
+impl Instruction for CustomInstructionBox {}
```

Comment:
Please do same changes for `complex_isi.rs` (add `impl Instruction` and remove `fn into_instruction()`)

Reply:
weird, I thought I did